core/lib/i18n/i18n.js

```diff
@@ -189,8 +194,12 @@ function createIcuMessageFn(filename, fileStrings) {
     const keyname = Object.keys(mergedStrings).find(key => mergedStrings[key] === message);
     if (!keyname) throw new Error(`Could not locate: ${message}`);
 
-    const filenameToLookup = keyname in fileStrings ? filename : getModulePath(import.meta);
-    const unixStyleFilename = path.relative(LH_ROOT, filenameToLookup).replace(/\\/g, '/');
+    // `message` can be a UIString defined within the provided `fileStrings`, or it could be
+    // one of the common strings found in `i18n.UIStrings`.
+    const filenameToLookup = keyname in fileStrings ?
+      filename :
+      path.relative(LH_ROOT, getModulePath(import.meta));
```

Comment:
no real change - just duping _some_ work because code is clearer. It's obvious that `getModulePath` would always returned an absolute path; but that the `filename` passed in could be absolute or relative is not so clear. so...relativize in two places